ideas/058-mainnet.md

```diff
@@ -86,14 +86,26 @@ Goal Date: 2018-05-31
 - security issues are tracked with `security` label in [status-react](https://github.com/status-im/status-react/issues?q=is%3Aissue+is%3Aopen+label%3Asecurity) and [status-go](https://github.com/status-im/status-go/issues?q=is%3Aissue+is%3Aopen+label%3Asecurity),
 - [a feature request](https://github.com/status-im/status-react/issues/3994) to allow custom upstream address was created
 
-## Status App Bug Bounty iteration (CANCELED)
+#### 2018-06-05
 
-**This iteration got canceled as we do not plan to announce Status App Bug Bounty for the time being.**
+- the final readout took place on May 7th,
+- there were no new issues reported,
+- all issues were fixed; there are still recommendations that we decided to implement but they are not blockers for beta. Moreover, more issues were identified during Risk Management meetings.
+
+## Status App Bug Bounty iteration ~~(CANCELED)~~
+
+~~**This iteration got canceled as we do not plan to announce Status App Bug Bounty for the time being.**~~
```

Comment:
Got resumed again right? Should we link to the blog post?

Also, WDYT about making the report public?

Reply:
Yeah, that's why I stroke it. I added information in the update below. There is no blog post yet but we can update it when it's published.

> Also, WDYT about making the report public?

I guess we are allowed to do that. We fixed all found issues I believe so that should not be a blocker. There are still some recommendations that we haven't worked on yet, though. The question is how we want to do it -- through a blog post or just on Slack?